### PR TITLE
CPP-298 Wait for approval on CircleCI builds from nori and renovate PRs 

### PR DESCRIPTION
**Pull request by emortong**
Renovate and Nori create many PRs which queue loads of builds in CircleCI blocking other PRs across the entire organisation from building, as a solution we are having PRs created by Renovate and Nori pause from building until approved. <br/><br/>This PR was created using a nori script 🍙<br/><br/>_Nori is a command-line application for managing changes across multiple (usually Github) repositories._

### CODE REVIEW

**Review by andygout — APPROVED** [review]
Before rolling out are you going to test this by creating a branch for this repo via Nori?

I was also wondering if we should be using `+` (1 or more of the preceding character) rather than `*` (0 or more of the preceding character) as you would not expect nameless branches, but I can see it's a convention already used elsewhere within the `.circle/config.yml` file so consistency is probably best.

**Comment by emortong** [comment]
> Before rolling out are you going to test this by creating a branch for this repo via Nori?
> 
@andygout This PR already applies the changes :) [Here](https://app.circleci.com/pipelines/github/Financial-Times/eslint-config-next?branch=nori%2FCPP-298-renovate-nori-wait-for-approval-7) is the link for the CircleCI build

>I was also wondering if we should be using + (1 or more of the preceding character) rather than * (0 or more of the preceding character) as you would not expect nameless branches, but I can see it's a convention already used elsewhere within the .circle/config.yml file so consistency is probably best.

Ok, I'll wait for Kara to log on and review before merging, to see if she has anything to say on this

Thank you!

**Review by apaleslimghost — APPROVED** [review]
this looks good to me! i don't think `*` vs `+` is going to make a difference in practice, like you said @andygout in real life there's not going to be any empty branch names, so it doesn't matter

**Pull request merged**
